packages/eslint-plugin-odyssey/src/rules/no-invalid-theme-properties.js

```diff
@@ -11,7 +11,7 @@
  */
 
 const path = require("path");
-const validThemeProperty = /^[A-Z][A-Za-z]+$/;
+const validThemeProperty = /^[A-Z][A-Za-z0-9]+$/;
```

Comment:
Can we go further and ignore the declaration rule if it ends in a number too? That prevents the need to eslint escape all over

Reply:
Do you have a quick code suggestion here? If not, I'll create a ticket and take care of this after release along with cleaning up the escapes.